acapi.yaml

```diff
@@ -11776,6 +11776,289 @@ components:
+            subtotal: 5500
+            shipping_total: 0
+            handling_total: 0
+            tax_total: 0
+            grand_total: 5500
+            tax_rate: 0
+            additional_fees: 0
+            discounted_shipping_total: 0
+            region_markup_amount: null
+            non_shipping_discount: 0
+            gift_certificate_discount: null
+            coupon_code: ''
+            associated_order_id: null
+            shipping_estimate_city: ''
+            shipping_estimate_state_code: ''
+            shipping_estimate_postal_code: ''
+            shipping_estimate_country_code: ''
+            is_shipping_billed_to_account: false
+            customer_payment_method_id: null
+            payment_method_id: null
+            is_payment_cart_only: false
+            items:
+              - id: 439
+                product_id: 62
+                item_number: ''
+                quantity: 1
+                price: 5500
+                cost: 0
+                shipping_address_id: 39
+                item_thumbnail: /images/LG/_thumb_60_50PY2DR_bic.jpg
+                item_url: ''
+                item_name: 60PY2DR
+                warehouse_id: 0
+                parent_product_id: null
+                parent_cart_item_id: null
+                updated_at: '2022-01-07T20:38:27.573+05:00'
+                created_at: '2022-01-07T20:38:27.573+05:00'
+                cart_id: 128
+                is_subscription_product: true
+                vendor_store_id: null
+                fitment: ''
+                configuration: ''
+                variants: null
+                personalizations: null
+            lookup_key: Tr0anQUb2yM%3D
+          customer:
+            id: 1
+            customer_number: ''
+            last_name: Barlow
+            first_name: Stefan
+            email: stefan@americomlp.com
+            phone_number: ''
+            registered_at: '2004-10-08T00:00:00+05:00'
+            last_visit_at: '2005-09-08T13:48:55+05:00'
+            adcode: ''
+            adcode_id: 0
+            affiliate_id: 0
+            customer_type_id: 1
+            is_no_tax_customer: false
+            comments: ''
+            store_id: 1
+            source: ''
+            search_string: ''
+            no_account: false
+            sales_person: ''
+            alternate_phone_number: ''
+            is_affiliate_customer: false
+            updated_at: '2022-03-04T18:21:45.413+05:00'
+            created_at: null
+            username: ''
+            is_contact_information_only: false
+            tax_exemption_number: ''
+            company: ''
+            source_group: ''
+            sales_person_user_id: null
+            store_payment_methods_enabled: []
+            tax_rate: null
+            reward_tier_id: null
+            sub: ''
+          shipping_address:
+            id: 267
+            customer_id: 1
+            company: Cart
+            address_line_1: mckldmcdm
```

Comment:
Can you change any referecence of 'mckldmcdm' to '123 Test' or something similar?

Reply:
yes Sure, Done